js/forms.js

```diff
@@ -279,85 +279,181 @@
      * Auto complete plugin  *
      *************************/
     $.fn.autocomplete = function (options) {
-      // Defaults
       var defaults = {
-        data: {}
-      };
+        data: {},
+        maxElementsAmount: null
+      },
+      currentLi = 0,
+      autocompleteOption = null,
+      currentAutocompleteVal = '';
 
       options = $.extend(defaults, options);
 
-      return this.each(function() {
-        var $input = $(this);
-        var data = options.data,
-            $inputDiv = $input.closest('.input-field'); // Div to append on
+      // Set input value
+      function setIputValue(ulElement, inputElement) {
+        ulElement.on('click', 'li', function () {
+          inputElement.val($(this).text().trim());
+          ulElement.empty();
+        });
+      }
+
+      // highlight elements
+      function highlight(string, $el) {
+        var img = $el.find('img');
+        var matchStart = $el.text().toLowerCase().indexOf("" + string.toLowerCase() + ""),
```

Comment:
it's a legacy after old code. I changed that and I'm hope so it's not a some kind of hack for old IE or something.